hosts

```diff
@@ -1,4 +1,7 @@
 [all]
 localhost ansible_connection=local
 
+[aws]
+ec2-54-148-68-112.us-west-2.compute.amazonaws.com
```

Comment:
This should not be required.  You should be able to derive this information from the dynamic inventory provided by ec2.py.